go/gas-oracle/oracle/gas_price_oracle.go

```diff
@@ -196,15 +196,15 @@ func NewGasPriceOracle(cfg *Config) (*GasPriceOracle, error) {
 	address := cfg.gasPriceOracleAddress
 	contract, err := bindings.NewGasPriceOracle(address, l2Client)
 	if err != nil {
-		return nil, err
+		return nil, wrapErr(err, "error creating contract binding")
```

Comment:
Do you mean to have gas_oracle changes in this PR?

Reply:
Nope will remove.